tests/test_parameter.py

```diff
@@ -74,19 +74,23 @@ def test_composed_param():
     param1 = Parameter('param1', 1.)
     param2 = Parameter('param2', 2.)
     param3 = Parameter('param3', 3., floating=False)
-    param4 = Parameter('param4', 4.)  # needed to make sure it does not only take all params as deps
+    param4 = Parameter('param4', 4.)  # noqa Needed to make sure it does not only take all params as deps
```

Comment:
```suggestion
    param4 = Parameter('param4', 4.)  # Needed to make sure it does not only take all params as deps
```

Reply:
noqa should be there, otherwise it's checked by linters as an unused parameter